Silicon/TigerlakePchPkg/Library/HeciInitLib/MkhiMsgs.h

```diff
@@ -31,10 +31,16 @@ typedef union {
 /// Defines for GroupID
 ///
 #define MKHI_CBM_GROUP_ID     0x00
+#define MKHI_FWCAPS_GROUP_ID  0x03
 #define MKHI_HMRFPO_GROUP_ID  0x05
 #define MKHI_MCA_GROUP_ID     0x0A
 #define MKHI_GEN_GROUP_ID     0xFF
 
+///
+/// Defines for FWCAPS_GROUP Command
+///
+#define FWCAPS_GET_RULE_CMD               0x02

```

Comment:
Just a nit. Want to align the value with the surrounding ones?

Reply:
sure, I can do that.

Reply:
fixed it in new commit.